services/inventory/grpc/services_server.go

```diff
@@ -36,10 +36,26 @@ func NewServicesServer(s *inventory.ServicesService) inventorypb.ServicesServer
 	return &servicesServer{s}
 }
 
+var serviceTypes = map[inventorypb.ServiceType]models.ServiceType{
+	inventorypb.ServiceType_MYSQL_SERVICE:      models.MySQLServiceType,
+	inventorypb.ServiceType_MONGODB_SERVICE:    models.MongoDBServiceType,
+	inventorypb.ServiceType_POSTGRESQL_SERVICE: models.PostgreSQLServiceType,
+	inventorypb.ServiceType_PROXYSQL_SERVICE:   models.ProxySQLServiceType,
+}
+
+func serviceType(serviceType inventorypb.ServiceType) *models.ServiceType {
+	if serviceType == inventorypb.ServiceType_SERVICE_TYPE_INVALID {
+		return nil
+	}
+	result := serviceTypes[serviceType]
+	return &result
```

Comment:
ServiceType is an int. 
If the function is called with an invalid parameter, what `return &result` would return?
The same way we return a nil for `serviceType == inventorypb.ServiceType_SERVICE_TYPE_INVALID`, if the function parameter is not valid, we should return a nil, maybe something like:
```
result, ok := serviceTypes[serviceType]
if !ok {
    return nil
}
return &result
```

Reply:
grpc should protect us from getting service types not declared in our proto files.
So if we missed any service type defined in proto files it's better to have panic message instead of silently going forward, IMHO.